packages/internal-test-env/index.ts

```diff
@@ -78,100 +77,96 @@ export interface TestingEnvironmentBase {
 type FeatureFlags = {
   [key: string]: any;
 };
+const ENV_VAR_PREFIX = "E2E_TEST_FEATURE_";
```

Comment:
This might be clearer as, though change isn't needed
```suggestion
const ENV_FEATURE_PREFIX = "E2E_TEST_FEATURE_";
```

Reply:
ENV_FEATURE_PREFIX